package.json

```diff
@@ -12,11 +12,12 @@
     "node": ">=10"
   },
   "dependencies": {
+    "@jest/globals": "^26.6.2",
```

Comment:
Out of curiosity isn't `jest` another react testing library? Is this a needed import? Is there something like this already in `@testing-library/react`?

Reply:
Jest, while initially built to support react testing, is not another react testing library. The react testing library we are currently using only supports component testing and not unit tests like the ones I wrote (at least to my knowledge, correct me if I'm wrong).

Reply:
I took a look at package.json and it looks like we already have Jest capabilities.